### DISCUSSION TITLE
File Details in the "Preview" Pane

### DISCUSSION THREAD

**Discussion by SpyderZT** [Flow-Launcher/Flow.Launcher#2232] [Ideas]
I know it's still in its infancy, but I'm hoping the preview pane will also display file details (Configurable) for the Explorer Plugin. Specifically I'm often wanting to see Size and Created / Modified / Accessed Dates, but I'm sure others might find use for other Metadata (Media Tags, Length, Etc.) as well. Is this in the pipeline?

**Comment by trACEroam** [comment]
i created a somewhat similar thread, I'm using explorer plugin with everything v1.5. I'd like to see the document and photo tags alongside the path while hovering search results

**Comment by SpyderZT** [comment]
Well I'm happy to see that the Size / Created / Modified info has been added in a recent(ish? And I just didn't notice it?) update! Allowing the rest of the information from the "Details" pane to be options for those who want them would be awesome, but that's the primary use I was looking for.
